dockerfileNative doesn't work with Gradle 8

### Expected Behavior

_No response_

### Actual Behaviour

```
> Error while evaluating property 'instructions.$8.text' of task ':api:dockerfileNative'.
    > Converting files to a classpath string when their paths contain the path separator ':' is not supported. The path separator is not a valid element of a file path. Problematic paths in 'file collection' are: '/home/app/resources:/home/app/application.jar'. Add the individual files to the file collection instead. Consult the upgrading guide for further information: https://docs.gradle.org/8.0.1/userguide/upgrading_version_7.html#file_collection_to_classpath

Caused by: org.gradle.api.GradleException: Converting files to a classpath string when their paths contain the path separator ':' is not supported. The path separator is not a valid element of a file path. Problematic paths in 'file collection' are: '/home/app/resources:/home/app/application.jar'. Add the individual files to the file collection instead. Consult the upgrading guide for further information: https://docs.gradle.org/8.0.1/userguide/upgrading_version_7.html#file_collection_to_classpath
        at org.gradle.internal.deprecation.DocumentedFailure$Builder.build(DocumentedFailure.java:69)
        at org.gradle.internal.deprecation.DocumentedFailure$Builder.build(DocumentedFailure.java:60)
        at org.gradle.api.internal.file.AbstractFileCollection.showGetAsPathDeprecationWarning(AbstractFileCollection.java:197)
        at org.gradle.api.internal.file.AbstractFileCollection.getAsPath(AbstractFileCollection.java:176)
        at org.graalvm.buildtools.gradle.internal.NativeImageCommandLineProvider.buildClasspathString(NativeImageCommandLineProvider.java:173)
        at org.graalvm.buildtools.gradle.internal.NativeImageCommandLineProvider.asArguments(NativeImageCommandLineProvider.java:113)
        at io.micronaut.gradle.docker.NativeImageDockerfile.buildNativeImageCommandLineArgs(NativeImageDockerfile.java:540)
        at io.micronaut.gradle.docker.NativeImageDockerfile.buildActualCommandLine(NativeImageDockerfile.java:515)
        at io.micronaut.gradle.docker.NativeImageDockerfile.lambda$setupInstructions$6(NativeImageDockerfile.java:432)
```

### Steps To Reproduce

Run `./gradlew dockerfileNative`

### Environment Information

- Gradle 8.0.1
- Micronaut 3.7.2
- Micronaut plugin 3.7.2

### Example Application

_No response_

### Version

3.7.2

--- a/docker-plugin/src/main/java/io/micronaut/gradle/docker/NativeImageDockerfile.java
+++ b/docker-plugin/src/main/java/io/micronaut/gradle/docker/NativeImageDockerfile.java
@@ -578,7 +578,11 @@ public abstract class NativeImageDockerfile extends Dockerfile implements Docker
         options.getConfigurationFileDirectories().setFrom(
                 remappedConfigDirectories
         );
-        options.getClasspath().from(getTargetWorkingDirectory().map(d -> d + "/libs/*.jar"), getTargetWorkingDirectory().map(d -> d + "/resources:" + d + "/application.jar"));
+        options.getClasspath().from(
+                getTargetWorkingDirectory().map(d -> d + "/libs/*.jar"),
+                getTargetWorkingDirectory().map(d -> d + "/resources"),
+                getTargetWorkingDirectory().map(d -> d + "/application.jar")
+        );
         options.getImageName().set("application");
     }
 

